content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -129,6 +129,24 @@ cloudprovider_gce_api_request_duration_seconds { request = "detach_disk"}
 cloudprovider_gce_api_request_duration_seconds { request = "list_disk"}
 ```
 
+
+### kube-scheduler metrics
+
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
```

Comment:
Please check to see if this needs to be added to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

If it does, please don't forget to add a description as well.

Reply:
This does not add a new feature gate, the gating is controlled by the existing metrics stability guarantees